index.ts

```diff
@@ -219,7 +226,7 @@ export type IId = string;
  * The storage keys of all the storage slots being requested
  */
 export type TStorageKeys = string[];
-export interface ITransaction {
```

Comment:
seems kinda weird that this would change. `I` is for interface, where as `T` is for a native type alias no?